src/main/kotlin/nl/tudelft/ewi/se/ciselab/testgenie/toolwindow/TestGenieToolWindowFactory.kt

```diff
@@ -28,13 +27,6 @@ class TestGenieToolWindowFactory : ToolWindowFactory {
         )
 
         val content: Content = contentFactory.createContent(quickAccessParameters.getContent(), "Parameters", false)
-
-        val visualisationService = project.service<CoverageToolWindowDisplayService>()
```

Comment:
I am a bit confused as to why you deleted that and where can I find the tab for the coverage visualization. If I pulled the latest changes, then I do not see the tab in the tool window.

Reply:
Seems like Qodana is no longer relevant, you fixed it while I was reviewing the old version. Thank you.